Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM maven:3.8.7-eclipse-temurin-17 AS build
+COPY . /home/app
+WORKDIR /home/app
+RUN mvn -B -U clean package -pl bpdm-certificate-management -am -DskipTests
```

Comment:
-pl is not needed as you don't have a multi module project

Reply:
Done